.github/workflows/daily-test-build.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-11, windows-latest]
-        libtiledb_version: ["dev", "release-2.14"]
+        libtiledb_version: ["dev", "release-2.14", "release-2.15"]
```

Comment:
Do we need to test against `release-2.14` anymore?

Reply:
Will remove in a week or two, after 2.15 is out.